location/gql_mutations.py

```diff
@@ -290,6 +293,11 @@ class Input(HealthFacilityInputType):
     @classmethod
     def async_mutate(cls, user, **data):
         try:
+            current_HF = HealthFacility.objects.get(uuid=data['uuid'])
+            if current_HF.code != data['code']:
```

Comment:
You're referencing data['code'] 2 times in here, can you put in in separate variable?